src/locales/it.ts

```diff
@@ -62,247 +62,253 @@ export default {
+    teresa_of_jesus_of_avila_virgin: 'Santa Teresa d’Avila, vergine e dottore della Chiesa',  // TODO: Localise as `Saint Teresa of Jesus of Ávila, Virgin and Doctor of the Church`
+    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin: 'Santa Teresa di Gesu’ Bambino, vergine e dottore della Chiesa',
+    thomas_aquinas_priest: 'San Tommaso d’Aquino, sacerdote e dottore della Chiesa',
+    thomas_becket_bishop: 'San Tommaso Becket, vescovo e martire',
+    thomas_apostle: 'San Tommaso, apostolo',
+    turibius_of_mogrovejo_bishop: 'San Turibio de Mogrovejo, vescovo',
+    vincent_of_saragossa_deacon: 'San Vincenzo, diacono e martire',
+    vincent_de_paul_priest: 'San Vincenzo de’ Paoli, sacerdote',
+    vincent_ferrer_priest: 'San Vincenzo Ferrer, sacerdote',
+    wenceslaus_i_of_bohemia_martyr: 'San Venceslao, martire',
+    seven_holy_founders_of_the_servite_order: 'Santi Sette Fondatori dei Servi della Beata Vergine Maria',
+    sunday_of_the_word_of_god: 'Domenica della Parola di Dio',
+    passion_of_saint_john_the_baptist: 'Martirio di San Giovanni Battista',
+    most_holy_name_of_jesus: 'Santissimo Nome di Gesú',
+    visitation_of_mary: 'Visitazione della Beata Vergine Maria',
+    our_lady_of_sorrows_principal_patroness_of_slovakia: 'Beata Vergine Maria Addolorata, patrona principale della Slovacchia',
+    george_of_lydda_martyr_principal_patron_of_england: 'San Giorgio, martire e patrono principale d’Inghilterra',
+    benedict_of_nursia_abbot_principal_patron_of_europe: 'San Benedetto, abate e patrono principale d’Europa',
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_secondary_patrons_of_europe: 'Santi Cirillo, monaco e Metodio, vescovo, patroni d’Europa',
+    teresa_benedicta_of_the_cross_stein_virgin_secondary_patron_of_europe: 'Santa Teresa Benedetta della Croce, vergine, martire e patrona secondaria d’Europa',
+    andrew_apostle_principal_patron_of_scotland: 'Sant’Andrea, apostolo e patrono principale della Scozia',
+    bridget_of_sweden_religious_secondary_patroness_of_europe: 'Santa Brigida, religiosa e patrona secondaria d’Europa',
+    catherine_of_siena_virgin_secundary_patroness_of_europe: 'Santa Caterina da Siena, vergine, dottore della Chiesa e patrona secondaria d’Europa',
+    catherine_of_siena_virgin_secondary_patroness_of_italy_and_europe: 'Santa Caterina da Siena, vergine, dottore della Chiesa e patrona secondaria d’Italia e d’Europa',
+    patrick_of_ireland_bishop_principal_patron_of_ireland: 'San Patrizio, vescovo e patrono principale dell’Irlanda',
+    columba_of_iona_abbot_secondary_patron_of_ireland: 'San Colombano, abate e patrono secondario dell’Irlanda',
+    our_lady_of_guadalupe_principal_patroness_of_the_americas: 'Beata Vergine Maria di Guadalupe, Patrona principale delle Americhe',
+    francis_of_assisi_principal_patron_of_italy: 'San Francesco d’Assisi, patrono principale d’Italia',
```

Comment:
@gtortone, could you proof the Italian patronage titles please? Below I list them all. Thanks in advance. :smiley:

```js
teresa_of_jesus_of_avila_virgin: 'Santa Teresa d’Avila, vergine e dottore della Chiesa',  // TODO: Localise as `Saint Teresa of Jesus of Ávila, Virgin and Doctor of the Church`
our_lady_of_sorrows_principal_patroness_of_slovakia: 'Beata Vergine Maria Addolorata, patrona principale della Slovacchia',
george_of_lydda_martyr_principal_patron_of_england: 'San Giorgio, martire e patrono principale d’Inghilterra',
benedict_of_nursia_abbot_principal_patron_of_europe: 'San Benedetto, abate e patrono principale d’Europa',
cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_secondary_patrons_of_europe: 'Santi Cirillo, monaco e Metodio, vescovo, patroni d’Europa',
teresa_benedicta_of_the_cross_stein_virgin_secondary_patron_of_europe: 'Santa Teresa Benedetta della Croce, vergine, martire e patrona secondaria d’Europa',
andrew_apostle_principal_patron_of_scotland: 'Sant’Andrea, apostolo e patrono principale della Scozia',
bridget_of_sweden_religious_secondary_patroness_of_europe: 'Santa Brigida, religiosa e patrona secondaria d’Europa',
catherine_of_siena_virgin_secundary_patroness_of_europe: 'Santa Caterina da Siena, vergine, dottore della Chiesa e patrona secondaria d’Europa',
catherine_of_siena_virgin_secondary_patroness_of_italy_and_europe: 'Santa Caterina da Siena, vergine, dottore della Chiesa e patrona secondaria d’Italia e d’Europa',
patrick_of_ireland_bishop_principal_patron_of_ireland: 'San Patrizio, vescovo e patrono principale dell’Irlanda',
columba_of_iona_abbot_secondary_patron_of_ireland: 'San Colombano, abate e patrono secondario dell’Irlanda',
our_lady_of_guadalupe_principal_patroness_of_the_americas: 'Beata Vergine Maria di Guadalupe, Patrona principale delle Americhe',
francis_of_assisi_principal_patron_of_italy: 'San Francesco d’Assisi, patrono principale d’Italia',
```

Reply:
ok, for me !

Reply:
Thanks! :smiley:

Reply:
@gtortone, I have just added another patronage title: `patron of Poland`. Is `patrono della Polonia` correct translation? Thanks in advance! :smiley:

Reply:
@tukusejssirs yes, "patrono della Polonia" is "patron of Poland" !